Makefile

```diff
@@ -56,6 +57,11 @@ $(STATIC_LIBS_DIR)/backbone.js: $(STATIC_LIBS_DIR)
 
 LIBS += $(STATIC_LIBS_DIR)/backbone.js
 
+$(STATIC_LIBS_DIR)/offline.min.js: $(STATIC_LIBS_DIR)
+	wget https://raw.githubusercontent.com/HubSpot/offline/v$(OFFLINE_VERSION)/offline.min.js -O $@
```

Comment:
Kind of a minor point but the other JS files are not the minified versions since they are easier to debug locally and we are going to minify via django-compressor anyway.